src/Pages/JoonWooPark/Main/Main.js

```diff
@@ -1,9 +1,82 @@
+import Nav from '../../../Components/Nav';
 import React from 'react';
+import StoryList from '../Components/StoryList';
+import jsonData from '../userData.json';
+import RecommendUser from '../Components/RecommendUser';
 import './Main.scss';
+import Feed from '../../../Components/Feed';
 
 class Main extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      currentUser: '',
+      userData: [],
+    };
+  }
+
+  componentDidMount() {
+    this.addData();
+  }
+
+  addData = () => {
+    fetch('http://localhost:3000/data/ParkJoonWoo/commentData.json')
+      .then(res => res.json())
+      .then(() => {
+        this.setState({ userData: jsonData });
+      });
+  };
+
   render() {
-    return <div>test</div>;
+    const { commentList, userData } = this.state;
+    return (
+      <div className="ParkJoonWoo">
+        <Nav />
+        <main>
+          <section className="left-container">
+            <div className="story-container">
+              {userData.length > 0 && <StoryList userData={userData} />}
```

Comment:
- state 에 초기값으로 바로 jsonData 를 할당하면 조건부 렌더링을 할 필요가 없어집니다. 
```suggestion
              <StoryList userData={userData} />
```